docs/index.rst

```diff
@@ -51,13 +51,15 @@ Download `Bebo for iOS <https://itunes.apple.com/us/app/bebo-private-server-for-
    database_api
    notification_api
    camera_api
+   audio_api
    misc_api
 
 .. toctree::
    :maxdepth: 2
-   :caption: Libraries
+   :caption: Modules
```

Comment:
the react lib is a lib, not a module. either create a new section called Modules (i'd suggest renaming that to something more conventional; hard to understand what it means right away as a newcomer), or put the leaderboard module somewhere else in the docs. but don't hide the react lib in an obscure section called modules